### PR TITLE
Adding error handling for tablets

### PR DESCRIPTION
**Pull request by goyalvarun601**
@kdimatteo

### CODE REVIEW

File: dist/jquery.audio-plugin.js [hunk 1]
```diff
@@ -109,12 +128,10 @@
   };
 
   MHEAudio.prototype.togglePlayback = function(){
-    if(this.isPlaying === false){
-      var _this = this;
-      var ua    = window.navigator.userAgent;
-      var isTablet = ua.indexOf('iPad') !== -1 || ua.indexOf('Android') !== -1;
+    var _this = this;
```
Comment (kdimatteo) [inline dist/jquery.audio-plugin.js]: do we need this scope reference?

**Comment by kdimatteo** [comment]
Please add tests if you have cycles.

File: dist/jquery.audio-plugin.js (hunk 1 above)
Comment (goyalvarun601) [inline dist/jquery.audio-plugin.js]: we need it inside the interval

**Comment by goyalvarun601** [comment]
I'll work on the tests, the existing tests are also broken.

**Comment by goyalvarun601** [comment]
Found a way to get the data object in the tests. Added two new tests for error handling.

**Pull request merged**